src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -129,7 +129,7 @@ export const TranscriptsListItemInfo = (
   };
   return (
     <div className={mainStyles}>
-      <div className={transcriptsListStyles.left}>bottom left</div>
+      <div className={transcriptsListStyles.left}></div>
```

Comment:
What's happening in this div? What's its purpose?

Reply:
This div will be fed with data (see screenshot) once available.
![Screenshot 2020-09-07 at 16 44 27](https://user-images.githubusercontent.com/6347854/92403482-72312d00-f129-11ea-9b85-a98244de66d8.png)